protogram/src/commonTest/kotlin/com/mattprecious/protogram/ProtogramTest.kt

```diff
@@ -14,7 +14,7 @@ class ProtogramTest {
     assertEquals(expected, actual)
   }
 
-  @Test fun `crazy nesting`() {
```

Comment:
Does this not work in multiplatform? I guess cause no jvm?

Reply:
Not on JS for sure. No idea what the native limitations are.